command/genesis/params.go

```diff
@@ -438,8 +444,9 @@ func (p *genesisParams) initGenesisConfig() error {
 
 	// burn contract can be set only for non mintable native token
 	if p.isBurnContractEnabled() {
-		chainConfig.Genesis.BaseFee = command.DefaultGenesisBaseFee
-		chainConfig.Genesis.BaseFeeEM = command.DefaultGenesisBaseFeeEM
+		baseFeeInfo, _ := parseBaseFeeConfig(p.baseFeeConfig)
```

Comment:
Consider logging an error message here and `p.baseFeeConfig` raw string, so that we are aware why we are running default configuration in case of some errors in `parseBaseFeeConfig`.

Reply:
@Stefan-Ethernal I have logged the error message  in `validateGenesisBaseFeeConfig`. 
Default configuration is only selected when `genesis-base-fee-config` flag is not used while genesis creation and when value of either baseFee or baseFeeEM is not provided in the flag.
Do you think is it still necessary to log error here?

Reply:
I see, can you just return provided `p.baseFeeConfig` string together with error when propagating an error from `validateGenesisBaseFeeConfig`?